content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,89 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
-   - **Mandatory** tasks can block a run from completing. If the task fails (including a timeout or unexpected remote error condition), a warning will be displayed on the run and the run will transition to an Errored state.
-
-
-![Screenshot: enforcement configuration for a specific task within a workspace](./images/run-tasks-add-to-workspace.png)
-
-Your run tasks are now configured. You can return to this page to configure or remove tasks in your workspace.
-
-![Screenshot: final configuration of tasks within a workspace](./images/run-tasks-final-workspace-configuration.png)
+Your run tasks are now configured. You can return to the settings page to configure or remove tasks in your workspace.
+    ![Screenshot: final configuration of tasks within a workspace](./images/run-tasks-final-workspace-configuration.png)
 
 
 ## Understanding Run Tasks Within a Run
 
-Workspaces with run tasks configured have an additional run stage called [Pre-Apply](../run/states.html). This stage is where your run tasks will execute during a run, after the plan stage and also the cost estimation and policy check stages, if applicable.
-
-The Pre-Apply stage will always end with the most restrictive status of the tasks configured to run. For example, if a mandatory task fails and an advisory task succeeds, the stage will fail and the run will error. If an advisory task fails but a mandatory task succeeds, the stage will succeed and the run will proceed to the apply stage.
+Run tasks execute between the [plan](../run/states.html#2-the-plan-stage) and [apply](../run/states.html#6-the-apply-stage) stage of a [Terraform run](../run/index.html). Once all of the run tasks executions are complete, the run will end based on the most restrictive enforcement level that has been configured in each of the associated run tasks.
 
-Regardless of the exit status of a task, the status and any related message will be displayed in the UI.
+For example, if a mandatory task fails and an advisory task succeeds, the run will fail. If an advisory task fails, but a mandatory task succeeds, the run will succeed and proceed to the apply stage. Regardless of the exit status of a task, the status and any related message data will be displayed in the UI.
 
-Here is an example of a run that failed due to a mandatory task.
+Here is an example of a run that failed due to a mandatory run task.
 
 ![Screenshot: a run with failed tasks](./images/run-tasks-run-failed.png)
 
 And here is an example of a run that succeeded.
 
-![Screenshot: a run with failed tasks](./images/run-tasks-run-success.png)
+![Screenshot: a run with passed tasks](./images/run-tasks-run-success.png)
+
+## Removing a Run Task from a Workspace
 
-## Deleting a Task from a Workspace
+1. Navigate to the desired workspace, then select "Run Tasks" from the "Settings" menu.
 
-1. Navigate to the desired workspace, then select "Tasks" from the "Settings" menu.
+2. Select the ellipses on the associated run task, and then select "Remove".
```

Comment:
@laurapacilio it would be great to get your take on the following. I'm not sure if this is the best way to phrase this,  and without an official style guide, I am a bit lost. 

I think it might be better to do the following but not sure if it's the right approach.
```
Select the ellipses (...) on the associated run task, and then select "Remove".
```

Reply:
hi! Yes I agree this is a tricky one. When our style guide doesn't cover something, we use google developers style guide. This is what it has to say about this: https://developers.google.com/style/ellipses#ellipses-in-a-user-interface

"When ellipses appear in a user interface, exclude them from the documentation describing the user interface unless their omission could cause confusion. For example, if the text on the button in the UI reads Save ..., document it as click Save."

This case is difficult because the button IS ellipsis. I talked to the education team, and we think that your approach with one tweak is the way we want to go. So:

"Click the ellipsis (...) button next to the Run Task, and then select **Remove**."

Also - our style guide now dictates that all UI elements should be bolded, rather than in quotes! Would you mind updating this PR to reflect that? We're slowly going to update all of the Cloud/Enterprise stuff to adhere to the new guideline.

Reply:
Thanks @laurapacilio I have updated the docs as requested. 

I have used `(...)` in the docs the first time that a user interacts with an ellipses button. I have left it out for any subsequent call outs in the docs.

Reply:
Thanks @laurapacilio I have updated the docs as requested. 

I have used `(...)` in the docs the first time that a user interacts with an ellipses button. I have left it out for any subsequent call outs in the docs.